examples/standalone-with-org-policies/README.md

```diff
@@ -0,0 +1,120 @@
+# Standalone With Org Policies Example
```

Comment:
consider changing the name of the example to something like `additional` or `enhanced` org policies.  "with" makes it sound like the blueprint main module doesn't have any org policies.

Reply:
I changed it to standalone-enhanced-org-policy.